android/src/main/java/cz/ackee/extensions/android/ViewExtensions.kt

```diff
@@ -15,6 +15,7 @@ import androidx.annotation.IntRange
 
 // Extensions for [View] class
 
+@Deprecated(message = "Use [View.isVisible] extension from core-ktx")
```

Comment:
what about usage of `ReplaceWith`? It would be ideal if we could just refactor this in the whole project because of that. You can see it in some other place in the repository

Reply:
it will not include core-ktx dependency automatically, so I thought that this is not necessary

Reply:
it is, because the app should have this dependency

Reply:
e5fbdab97a2ca5360862fc4ae054101a45662f70